### DISCUSSION TITLE
The online page can't work

### DISCUSSION THREAD

**Discussion by moyi-dong** [tilk/digitaljs_online#63] [General]
https://digitaljs.tilk.eu/
It seems doesn't work, perhaps the backend server needs to be fixed.

**Comment by pedrobonifacio** [comment]
I need this fix to complete my homework. Can anyone help me with an alternative?

**Comment by yanghuaxuan** [comment]
> POST https://digitaljs.tilk.eu/api/yosys2digitaljs 504 (Gateway Time-out)

It seems that the API for retrieving yosys2digitaljs library is down right now.

**Comment by moyi-dong** [comment]
> I need this fix to complete my homework. Can anyone help me with an alternative?

I deployed his webapp version at https://github.com/tilk/digitaljs_online/  you can use it locally. If you need further information , feel free to contact me.

**Comment by tilk** [comment]
I've restarted the service, somehow it was alive but not responding. I currently don't have the time to debug this; maybe I'll create some kind of watchdog to automatically restart if this happens again. Still, I can't guarantee this won't happen again.

If you are using the service for a class, please tell your teacher to host his own instance. Mine is provided free as a courtesy, I don't provide any service or warranty for it.
